src/main/js/components/table/tableNavigationWorker.js

```diff
@@ -101,7 +102,17 @@ export function getKeyboardShortcuts() {
       );
     },
     text: (event) => {
-      if (selectedCell && !selectedCellEditing
+      if (selectedCell && event.ctrlKey && event.key === "c"  // Cell copy
```

Comment:
Could someone on Mac please test this? I'll try to figure out code for Cmd-keys and simplest OS-detection meanwhile.